menu.php

```diff
@@ -200,16 +200,16 @@
 			<div id="collapseSeven" class="accordion-collapse collapse" aria-labelledby="sides" data-bs-parent="#accordianMenu">
 				<div class="accordion-body">
 					<ul class="list-group">
-						<li class="list-group-item">Eggs :Scrambled,Over Easy,Sunny Side,Boiled</li><br />
-						<li class="list-group-item">Great Grandads Grits</li><br />
+						<li class="list-group-item">Eggs: Scrambled, Over Easy, Sunny Side, Boiled</li><br />
+						<li class="list-group-item">Great Grandad's Grits</li><br />
```

Comment:
If you add a comma after Great, this would be hilarious but grammatically a disaster lolol

Reply:
LMAO! I am rejecting this comment due to my neuroticism.

Reply:
@eiman-kased lol!!!